test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -57,6 +61,27 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 	f := framework.NewDefaultFramework("graceful-node-shutdown")
 	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
 
+	ginkgo.BeforeEach(func() {
+		systemdVersion := 0
+		if _, err := exec.LookPath("systemd-run"); err == nil {
+			if version, verr := exec.Command("systemd-run", "--version").Output(); verr == nil {
+				// sample output from $ systemd-run --version
+				// systemd 245 (245.4-4ubuntu3.13)
+				re := regexp.MustCompile(`systemd (\d+)`)
+				if match := re.FindSubmatch(version); len(match) > 1 {
+					systemdVersion, _ = strconv.Atoi(string(match[1]))
+				}
+			}
+		} else {
+			framework.Logf("unable to find systemd-run in path : %v", err)
```

Comment:
should we skip test in this case? I'd say let's run it and if it's failing on some system recommend to add systemd-run to path

Reply:
sure i can switch over to just run it